common/src/main/scala/explore/components/HelpIcon.scala

```diff
@@ -24,17 +26,23 @@ object HelpIcon {
 
   implicit def render(props: HelpIcon): VdomElement = component(props).vdomElement
 
-  val component = ScalaFnComponent[Props] { p =>
-    HelpCtx.usingView { help =>
-      val helpMsg = help.zoom(HelpContext.displayedHelp)
-      <.span(
-        ^.onClick ==> { (e: ReactMouseEvent) =>
-          e.stopPropagationCB *> e.preventDefaultCB *> helpMsg.set(p.id.some)
-        },
-        Icons.Info
-          .fixedWidth()
-          .clazz(ExploreStyles.HelpIcon)
-      )
+  implicit val helpReusability: Reusability[HelpIcon] =
+    Reusability.by(x => (x.id.value, x.clazz.map(_.htmlClass)))
+
+  val component = ScalaFnComponent
+    .withHooks[Props]
+    .renderWithReuse { p =>
```

Comment:
Since there are no hooks defined, I'm guessing it's not possible to use `renderWithReuse` if you just do `ScalaFnComponent[Props]` ?

Reply:
That's right, you can't get `renderWithReuse` unless you have hooks

Reply:
We should probably suggest supporting this to japgolly.

Reply:
Yes, but don't know how to say. The only think I know about `hooks` is they exist in react